Remove availability

msdemlei opened the pull request:
This PR largely removes the notion of the VOSI availability endpoint, and at least the requirement to have them. The PR contains text that discusses the reasons (see also http://mail.ivoa.net/pipermail/dal/2021-June/008444.html, and we had an Interop session where nobody wanted to speak up for it).

I've taken the liberty of adding a few unrelated TODO-s while going through the text. I'll take them out again if people feel they're spoiling things, but making separate PRs for this kind of thing would suck, too. Yes, we could to github issues, but, frankly, I'd much rather have this kind of thing in the interoperable git history than in the proprietary github issues... So... if you can, bear with me.

msdemlei committed:
Unrelated cleanup: Removing legacy README.

msdemlei committed:
Removing spec and requirement for availability

msdemlei committed:
Adding a StandardsRegExt record for VOSI.

Well, it turns out there hasn't been one all these years, which means that
all VOSI StandardIDs ("ivo://ivoa.net/std/VOSI#capabilities") really pointed
nowhere.

That nobody even noticed may raise questions on how useful StandardsRegExt
StandardKeys actually are.  But that's beside the point here.

I need the record here to say something about what #availability is
in a post-caproles VO.

mbtaylor commented:
The VOSICapabilities schema, as well as the VOSIAvailability schema, has disappeared from the Appendices.  Is this intentional?

msdemlei committed:
Restoring VOSICapabilities in the printed document.

Also changing the inclusion of the VOSITables schema to lstinputlisting;
pulling this from the authoritative, on-disk version seems a good idea.

Whether it's a good idea to print the two schema files at all is another
matter.  I'd say no, but I'd leave removing them to the editor.

msdemlei committed:
Updating Makefile to auto-pull ivoatex if necessary.

Also including git metadata.

msdemlei commented:
On Wed, Feb 23, 2022 at 05:56:41AM -0800, Mark Taylor wrote:
> The VOSICapabilities schema, as well as the VOSIAvailability
> schema, has disappeared from the Appendices.  Is this intentional?

Good question, because I frankly think printing schemas is a waste of
paper, and we should rather use ivoatexDoc's auxiliaryurl to have
in-document references to the per-version schemas.

But no, the VOSICap removal was collateral damage; I'd say it's the
editor's call whether the schemas should remain in.  I've undone it, except
I'm not reading it (and VOSITable, as well) from the disk in hopes to
avoid version skews as we touch these.

pdowler commented on VOSI.tex:
```diff
@@ -394,29 +363,22 @@ \subsection{Non-service metadata (non-normative)}
 capabilities & \nolinkurl{ivo://ivoa.net/std/VOSI#capabilities} \\
 tables (1.0) & \nolinkurl{ivo://ivoa.net/std/VOSI#tables} \\
-tables (1.1) & \nolinkurl{ivo://ivoa.net/std/VOSI#tables-1.1} \\
+tables (1.1 and 1.2) & \nolinkurl{ivo://ivoa.net/std/VOSI#tables-1.1} \\
```
I am not certain that the #tables-1.1 key will apply to version 1.2 or not, so this may change but OK for now.

pdowler commented on VOSI.tex:
```diff
@@ -688,215 +641,17 @@ \subsection{Example 4: Child table resource}
-
 \section{The Complete VOSICapabilities Schema}
 \label{appendix:capabilities}
-\begin{lstlisting}[language=XML]
```
I agree to removing the xsd from the document.

pdowler commented:
there is no github CI to check it, so I will merge and add the standard CI in a separate PR.
